exercises/rotational-cipher/cannonical-data.json

```diff
@@ -0,0 +1,72 @@
+{
+    "#": [
+        "The tests are a series of rotation tests and a bonus test: ",
+        "* Rotational tests",
```

Comment:
I believe this line and the one below it can be removed.

actually, in that case, maybe the entire `"#"` section can be removed.